docs/proposals/wikilinks-in-foam.md

```diff
@@ -0,0 +1,97 @@
+# Wikilinks in Foam
+
+Foam supports standard wikilinks in the format `[[wikilink]]`.
+
+Wikilinks can refer to any note or attachment in the repo: `[[note.md]]`, `[[doc.pdf]]`, `[[image.jpg]]`.
+
+The usual wikilink syntax without extension refers to notes: `[[wikilink]]` and `[[wikilink.md]]` are equivalent.
+
+The goal of wikilinks is to uniquily identify a file in a repo, no matter in which directory it lives.
```

Comment:
Small typo here: uniquely.

Reply:
thanks! fixed in https://github.com/foambubble/foam/commit/59bb2eb38f08056867834cf2ee1ae432d68da67a